views/update-experimenter.jade

```diff
@@ -0,0 +1,37 @@
+doctype html
+html
+  head
+    title Fish - Update My Profile
+    meta(name='viewport', content='width=device-width, initial-scale=1.0')
+    link(rel='stylesheet', href='/bower/bootstrap/dist/css/bootstrap.min.css')
+    link(rel='stylesheet', href='/public/css/grid.css')
+    script(src='/bower/jquery/jquery.min.js')
+    script(type='text/javascript', src='/public/js/update-experimenter.js')
+  body
+    .container
+      .page-header
+        h1 Update Profile
+      .col-md-4  
+        .row
+          .form-group
+            label.form-label.col-md-6(for='name') Name
+            .col-md-6
+              input#name.form-control(type='text', name='name', autofocus='')
+        .row
+          .form-group
+            label.form-label.col-md-6(for='username') Username
+            .col-md-6
+              input#username.form-control(type='text', name='username')
+        .row
+          .form-group
+            label.form-label.col-md-6(for='email') Email
+            .col-md-6
+              input#email.form-control(type='email', name='email')
+        .row
+          .form-group
+            label.form-label.col-md-6(for='password') Password
+            .col-md-6
+              input#password.form-control(type='text', name='rawPassword')
+        button#do-update.btn.btn-info(type='submit') Update profile
```

Comment:
It's best to adhere to the convention in the rest of the system for the OK/Cancel buttons: OK (in this case, Update profile) on green, and Cancel without a background--see for instance the page for creating a new microworld.